data/data/baremetal/masters/variables.tf

```diff
@@ -33,3 +28,18 @@ variable "instance_infos" {
   type        = list(map(string))
   description = "Instance information for hosts"
 }
+
+variable "ignition_url" {
```

Comment:
Include "master" in the names of these variables to make it clear that they are for the masters and not the bootstrap.

Reply:
This one is already nested under masters directory, so can't we infer that these variables belong to the master? Besides, I think this change will impact the TF provider code as well.

Reply:
Sorry, I clicked the wrong file. I mean the ones [here](https://github.com/openshift/installer/pull/4427/files#diff-328dc01a4376155918bed799b815ab5731f7dec17ab3e8ea83a40b581fc6e018R56-R69)

Reply:
And [here](https://github.com/openshift/installer/pull/4427/files#diff-3b0c0219b1206253af5d430d464ea73c976c339a055510a0d9456d4dfb0419afR30-R32)

Reply:
I updated the terraform variable names (not in the TF provider only the installer tf files), but I missed the code variable names (that's not exposed to the user but I'll update for consistency)

Reply:
Now done